app/views/orga/seasons/_switch.html.erb

```diff
@@ -0,0 +1,17 @@
+<br/>
+<hr>
+<h3> For development only: select season phase… </h3>
+<p>… to adjusts all the date dependent settings for <i>current_season</i>.
+
+<%= form_tag orga_seasons_path, action: switch_phase, method: :get do %>
+    <%= radio_button_tag :option, "Application" %>
+     <%= label_tag :option_Application, "Application Phase" %> 
+    <%= radio_button_tag :option, "CodingSummer" %> 
+    <%= label_tag :option_CodingSummer, "Coding Summer" %> 
+    <%= radio_button_tag :option, "RealTime" %>
+    <%= label_tag :option_RealTime, "Back to Real Time" %>
+      <%= submit_tag("Switch phase") %>
+  <% end %>
+
+<p><%= " current_season set to #{params[:option]} " %><br>
```

Comment:
It wasn't meant for debugging, but as simple user action feedback. :-)  Already changed that into a nice flash notice. I left the season.inspect though.